CMakeLists.txt

```diff
@@ -289,21 +289,28 @@ endif()
 find_package(ZLIB REQUIRED)
 include_directories(${ZLIB_INCLUDE_DIRS})
 
+set(DOWNLOAD_SDL2 TRUE CACHE BOOL "Download the SDL2 headers and libraries.")
```

Comment:
```suggestion
set(DOWNLOAD_SDL2 TRUE CACHE BOOL "Download the SDL2 headers for compilation. Set to FALSE to use system headers.")
```

Reply:
we don't actually care about the libraries. they don't get installed with the DFHack package.